social_bot/social_bot.cc

```diff
@@ -34,6 +54,39 @@ class Agent {
     }
     return names;
   }
+
+  std::vector<std::string> GetSensorNames() {
+    std::vector<std::string> sensorNames;
+    gazebo::sensors::SensorManager* mgr =
+        gazebo::sensors::SensorManager::Instance();
+    sensorNames.reserve(mgr->GetSensors().size());
+
+    for (auto sensor : mgr->GetSensors()) {
+      sensorNames.push_back(sensor->Name());
```

Comment:
We use scoped name for accessing joint. But it's unscoped name for sensor? And since this is a member function for Agent, if possible, we should only return the sensors for this agent.